packages/web/examples/ErrorBoundary/src/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+	ReactiveBase,
+	DynamicRangeSlider,
+	SelectedFilters,
+	ResultList,
+	ReactiveList,
+	ErrorBoundary,
+} from '@appbaseio/reactivesearch';
+
+import './index.css';
+
+const Main = () => (
+	<ReactiveBase
+		app="good-books-ds"
+		url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@appbase-demo-ansible-abxiydt-arc.searchbase.io"
+		enableAppbase
+	>
+		<div className="row">
+			<div className="col">
+				<ErrorBoundary
```

Comment:
@SavvyShah ErrorBoundary should be used just after the `ReactiveBase`  so it can catch errors for the whole tree.